src/recipes/gin-sour.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Gin Sour",
+    "description": "This sour concoction may be a little too tart and bitter for some tastes, but the sour family is one of our favourite type of drinks and the Gin Sour sits at the top",
+    "github": "cemuka",
+    "ingredients": [
+        "50 ml Gin", 
+        "25 ml Lemon Juice", 
+        "12 1/2 ml Simple Syrup", 
+        "Splash Egg-White"
+    ],
+    "directions": [
+        "Begin by chilling glass with ice and soda water. Add the gin, simple syrup, lemon juice and egg white to a mixing tin and dry shake for 10 seconds.",
+        " Add ice to the mixing tin and shake for a further 10 seconds.",
+        "Fill glass with ice and strain drink into glass.",
+        "Squeeze some lemon zest over drink and add lemon twist into glass."
+    ],
+    "image": "<drink-name.jpg>"
```

Update image property to:
```suggestion
    "image": "gin-sour.jpg"
```